suggestions.md

```diff
@@ -0,0 +1,228 @@
+# Suggestions Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/suggestions**   
+
+_Unsupported._ The suggestions can be retrieved only by source and target or via direct link, see the single entry and search method below. 
+
+### single entry
+**GET api/integration/suggestions/<:suggestion-id>**
+
+It returns the data from one suggestion. The suggestion-id is a String generated by the source, as it must be unique for source, target and identified object to be imported it is usually represented by a combination of such elements
+
+sample for a suggestion /api/integration/suggestions/reciter:gf3d657-9d6d-4a87-b905-fef0f8cae26:24694772
```

Comment:
did you consider specifying the source as it is for authority or external sources?

` "href": "https://dspace7.4science.cloud/server/api/integration/authorities/authors/entryValues/d4b5ca88-9d6d-4a87-b905-fef0f8cae26c"`

this could be transformed into:
`/api/integration/suggestions/reciter/entryValues/gf3d657-9d6d-4a87-b905-fef0f8cae26:24694772`